src/routes/api.ts

```diff
@@ -1,9 +1,16 @@
 import { Request, Response, Router } from 'express'
 import {latestDigestedPostNumber} from '../controllers/mysql/queries';
-
+import {GetServerStatus, SetServerStatus } from '../controllers/serverstatus';
+import fetch from 'node-fetch';
+import { get } from 'https';
+import { runInNewContext } from 'vm';
 
 const router: Router = Router();
 
+//Set Server status upon startup
+SetServerStatus("Alive");
```

Comment:
Should not be in the API, should be in app.ts if anywhere (is it even neccesary)